test/e2e/tests/releases/displayDocument/DisplayedDocument.spec.ts

```diff
@@ -0,0 +1,445 @@
+import {expect} from '@playwright/test'
+import {type SanityDocument} from '@sanity/client'
+import {test} from '@sanity/test'
+
+import {
+  speciesDocumentNameASAP,
+  speciesDocumentNameDraft,
+  speciesDocumentNamePublished,
+  speciesDocumentNameUndecided,
+} from '../utils/__fixtures__/documents'
+import {
+  partialASAPReleaseMetadata,
+  partialScheduledReleaseMetadata,
+  partialUndecidedReleaseMetadata,
+} from '../utils/__fixtures__/releases'
+import {
+  archiveAndDeleteRelease,
+  createDocument,
+  createRelease,
+  discardVersion,
+  getRandomReleaseId,
+} from '../utils/methods'
+
+// for before all to work with single worker and run only once per tests and describe
+// this is to avoid issues with multiple releases being created per test
+test.describe.configure({mode: 'serial'})
+
+test.describe('displayedDocument', () => {
+  let publishedDocument: SanityDocument
+  let publishedWithVersion: SanityDocument
+  let publishedDocumentDupe: SanityDocument
+  let versionDocumentOne: SanityDocument
+  let versionDocumentTwo: SanityDocument
+
+  let asapReleaseId: string
+  let undecidedReleaseId: string
+
+  let multipleVersionsDocId: string
+  let singleASAPVersionDocument: SanityDocument
+
+  test.beforeAll(async ({sanityClient, _testContext, browserName}) => {
+    test.skip(browserName === 'firefox', 'Skip test outside of chromium due to flakiness')
```

Comment:
Since it's important that this logic is the same everywhere, maybe we can configure this in the global scope? As it is now It's a little too easy for comfort to accidentally skip a test, but forget to skip a cleanup function and vice versa. Suggestion:
```ts

// skip firefox due to flakyness
const SKIP_BROWSERS =['firefox']
```
and then instead update these conditions to say:
```ts
test.skip(SKIP_BROWSERS.includes(browserName), `Skip ${browserName} due to flakiness`)
```

Reply:
Good point, done